flytekit/core/data_persistence.py

```diff
@@ -52,6 +52,8 @@
 
 Uploadable = typing.Union[str, os.PathLike, pathlib.Path, bytes, io.BufferedReader, io.BytesIO, io.StringIO]
 
+_WRITE_SIZE_CHUNK_BYTES = int(os.environ.get("_F_P_WRITE_CHUNK_SIZE", "26214400"))  # 25 * 2**20
```

Comment:
Can you leave a comment? 

Also, why the `_` in the constant name?

Reply:
it's fairly internal to this file, thought the _ would be nice.  i'll add a comment.